### ISSUE TITLE
Invalid prop `flexWrap`

### ISSUE THREAD

**Issue by dmackerman** [jxnblk/grid-styled#127]
The following:
`<Flex flexWrap mb={2}>`

Results in: 

```
Warning: Failed prop type: Invalid prop `flexWrap` supplied to `Flex`.
    in Flex (at index.js:123)
```

The resulting DOM node does get the proper `flex-wrap: wrap` style applied, though.

**Comment by Whyseverythingtaken** [comment]
I get this warning too! Would be great to get a fix or a workaround

**Comment by jxnblk** [comment]
The `flexWrap` prop requires a string or array value 
https://github.com/jxnblk/grid-styled/blob/master/docs/Flex.md#props
